src/nl/rubensten/texifyidea/window/PreviewForm.kt

```diff
@@ -0,0 +1,126 @@
+package nl.rubensten.texifyidea.window
+
+import com.google.common.io.Files
+import java.io.*
+import java.util.concurrent.TimeUnit
+import javax.imageio.ImageIO
+import javax.swing.*
+
+class PreviewForm {
+    var panel: JPanel? = null
```

Comment:
Newline before panel.
```suggestion

    var panel: JPanel? = null
```